packages/office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlert.tsx

```diff
@@ -0,0 +1,163 @@
+/**
+ * @copyright Microsoft Corporation. All rights reserved.
+ *
+ * @file Screen Reader Alert component.
+ */
+
+import * as React from 'react';
+
+import { IScreenReaderAlertProps, defaultScreenReaderAlertProps, ReadingMode } from './ScreenReaderAlert.Props';
+import styles = require('./ScreenReaderAlert.scss');
+
+export interface IScreenReaderAlertState {
+  alertText: string;
+}
+
+/**
+ * The period of timeout used to clear the text after each rendering.
+ *
+ * More than 1000ms is critical for NVDA to read the text properly.
+ */
+const clearTextTime: number = 1000;
+
+/**
+ * This is a screen reader alert component for developers to easily add screen reader feature to their web site.
+ * This component is a wrapper, you would like to wrap the texts for screen reader to read.
+ * @see {@link https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions Live Region in MDN}
+ *
+ * All functionalities of this component are tested through:
+ * 1. Narrator in Windows 10.
+ * 2. JAWS 18.
+ * 3. NVDA 2016.4.
+ * 4. ChromeVOX v53. (Not support ReadingMode.ReadAfterOtherContent option for ChromeVOX, it will read immediately)
+ *
+ * @example
+ * <ScreenReaderAlert>
+ *   { yourFirstAlertText }
```

Comment:
Fixed, previous we were extracting string from React component, we removed that feature, it's definitely better to use text props now, thanks